ts/components/session/RegistrationTabs.tsx

```diff
@@ -713,13 +713,8 @@ export class RegistrationTabs extends React.Component<{}, State> {
   }
 
   private async resetRegistration() {
-    await window.Signal.Data.removeAllIdentityKeys();
-    await window.Signal.Data.removeAllPrivateConversations();
-    window.Whisper.Registration.remove();
```

Comment:
what does this do?

Reply:
This removes all keypairs and conversations as well as marks signals registration as not done

Reply:
to clarify just talking about 718, which I understand you have to said:
> marks signals registration as not done
Why do we no longer need that?

Reply:
`Registration.remove()` calls `storage.remove('chromiumRegistrationDone');`
so it's just removing a field from the storage, if we wipe the storage then it's the same behaviour